src/styleguide/analyzer/typescript-react-analyzer/typescript-react-analyzer.ts

```diff
@@ -77,6 +77,11 @@ export class Analyzer extends StyleguideAnalyzer {
 					program,
 					exportInfo.type
 				);
+
+				if (!reactType) {
+					return;
+				}
+
 				const propType = reactType ? reactType.getTypeArguments()[0] : undefined;
```

Comment:
Oh, I just see something new. The ternary check ("?:") is no longer needed here, right?